From 33718e98a0e0a0bca068c22d44e5f7b3ae4f1a3b Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Tue, 3 Jul 2012 17:18:08 +0800
Subject: ENGR00182743-4 V4L2 output: Add non-interleaved YUV444 pixel format
 support

Add non-interleaved YUV444 pixel format IPU_PIX_FMT_YUV444P support

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 drivers/media/video/mxc/output/mxc_vout.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/media/video/mxc/output/mxc_vout.c b/drivers/media/video/mxc/output/mxc_vout.c
index f1f376d..4b3617f 100644
--- a/drivers/media/video/mxc/output/mxc_vout.c
+++ b/drivers/media/video/mxc/output/mxc_vout.c
@@ -196,6 +196,10 @@ const static struct v4l2_fmtdesc mxc_formats[] = {
 		.description = "TILED NV12F",
 		.pixelformat = IPU_PIX_FMT_TILED_NV12F,
 	},
+	{
+		.description = "YUV444 planar",
+		.pixelformat = IPU_PIX_FMT_YUV444P,
+	},
 };
 
 #define NUM_MXC_VOUT_FORMATS (ARRAY_SIZE(mxc_formats))
-- 
1.7.9.5

